servers/alzirr/deployment.nix

```diff
@@ -0,0 +1,109 @@
+{ config, pkgs, lib, options, inputs, ... }:
+let
+  swampwalk-profile = "/nix/var/nix/profiles/per-user/buildkite-swampwalk/swampwalk";
+  swampwalk-frontend-profile = "/nix/var/nix/profiles/per-user/buildkite-swampwalk/swampwalk-frontend";
+
+in
+{
+  imports = [
+    ../.
```

Comment:
remove? I think it's just the defaults stuff, which is now split into an included common in both this repo and serokell.nix, though worth checking we don't need anything that we didn't copy yet